Dolapo47/Transport-App
# implement create a bus feature

What does this PR do?
A summary of what this task is about
This pull request lets the admin create a bus

Description of Task to be completed?
A list of the major tasks you did to complete this feature/bug/chore
set up the bus service
set up the create bus route

How should this be manually tested?
Outlined steps on how someone else can successfully test this feature/chore/bug to ensure it works as expected
register a new user on postman; retrieve the token from registered email; verify the email in the verify password route; then as an admin, create a bus
